DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -269,6 +270,56 @@
 mu8diEle12CaloIdLTrackIdL_dz.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_DZ_v*")
 mu8diEle12CaloIdLTrackIdL_dz.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_v*")
 
+###########AK8 jet trigger with AK8 CSV requirement############
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring = hltBTVmonitoring.clone()
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.FolderName = cms.string('HLT/Higgs/ggHbb/CSV04/')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.nmuons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.nelectrons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.njets = cms.uint32(1)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.jets = cms.InputTag("ak8PFJetsCHS")
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.jetSelection = cms.string('pt>300 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.bjetSelection = cms.string('pt>330 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_AK8PFJet330_PFAK8BTagCSV_p1_v*')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.jetPtBinning = cms.vdouble(0,300,310,330,350,370,400,450,500,700,1000,1500,3000)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.lsPSet = cms.PSet(
+  nbins = cms.uint32( 1 ),
```

Comment:
I'm a little bit confused :(
#nbins = 1 ......

Reply:
That's how I disabled the LS histograms. Reducing the bins from 250 to 1. Is there a better way to do this?

Reply:
0-level approach, I would add a flag

Reply:
Sorry, I'm unfamiliar with this. How would I do this?

Reply:
if the flag is not already available, then you should update the c++ code
by adding the flag (bool)
but, ok, 1 bin does not cost much, even if it is kind of quick-n-dirt